### PR TITLE
Remove experimental annotation from non-experimental classes.

### PR DESCRIPTION
**Pull request by arturdryomov**
The only remaining `@Experimental` consumer is `InstanceFactory`. Really not sure if we need it at all since I don’t see a lot of usage here. It is kind of superseded by `Root.registerListener` from my  point of view. WDYT?

Also moved `@Ignore` to `.meta`, but we need to rethink the packaging a bit (or not).

### CODE REVIEW

**Review by artem-zinnatullin — APPROVED** [review]

File: spek-dsl/common/src/main/kotlin/org/spekframework/spek2/style/gherkin/Gherkin.kt [hunk 1]
```diff
@@ -30,7 +30,7 @@ class Scenario(delegate: ActionBody) : ActionBody by delegate {
     }
 
     fun Then(description: String, body: TestBody.() -> Unit) {
-        test("Then $description", body = body)
+        test("And $description", body = body)
```
Comment (raniejade) [inline spek-dsl/common/src/main/kotlin/org/spekframework/spek2/style/gherkin/Gherkin.kt]: Accidental change?

**Review by raniejade — CHANGES_REQUESTED** [review]

File: spek-dsl/common/src/main/kotlin/org/spekframework/spek2/style/gherkin/Gherkin.kt (hunk 1 above)
Comment (arturdryomov) [reply spek-dsl/common/src/main/kotlin/org/spekframework/spek2/style/gherkin/Gherkin.kt]: Yep! Thanks, removed it.

**Review by raniejade — APPROVED** [review]
`InstanceFactory`is used in cases you want to control the classloader used to run the tests.

**Comment by arturdryomov** [comment]
> InstanceFactoryis used in cases you want to control the classloader used to run the tests.

@raniejade, yep, but in which cases do you want to control that?

**Comment by raniejade** [comment]
Robolectric tests for example.

**Pull request merged**